authorize.go

```diff
@@ -35,7 +35,7 @@ func (p PolicySet) IsAuthorized(entityMap Entities, req Request) (Decision, Diag
 	// - For permit, all permits must be run to collect annotations
 	// - For forbid, forbids must be run to collect annotations
 	for id, po := range p.policies {
-		result, err := po.eval.Eval(c)
+		result, err := po.eval.Eval(env)
```

Comment:
I question not passing Env by reference here, mostly because Env will get large, and that's likely to result in a lot of copying? I don't believe that Go has an optimization that can detect potentially large copies and silently convert to pass by reference, but I could be wrong. If so, then, this is fine.

Reply:
I forgot that Env is "Just" a tuple of pointers. This shouldn't be a big deal then. Nevermind.